src/main/java/frc/robot/hardware/interfaces/IAccelerometer.java

```diff
@@ -0,0 +1,13 @@
+package frc.robot.hardware.interfaces;
+
+public interface IAccelerometer {
+
+	double getAccelerationMagnitude();
+
```

Comment:
what about rotational acceleration?
Im not sure it's a feature but if it is we want is